clients/js/src/cmds/status.ts

```diff
@@ -33,34 +29,32 @@ export const builder = (y: typeof yargs) =>
 export const handler = async (
   argv: Awaited<ReturnType<typeof builder>["argv"]>
 ) => {
-  const network = argv.network.toUpperCase();
-  assertNetwork(network);
-  const chain = argv.chain;
+  const network = getNetwork(argv.network);
+  const chain = chainToChain(argv.chain);
   assertChain(chain);
 
-  const addr =
-    relayer.RELAYER_CONTRACTS[network][chain]?.wormholeRelayerAddress;
+  const addr = contracts.relayer.get(network, chain);
   if (!addr) {
     throw new Error(`Wormhole Relayer not deployed on ${chain} in ${network}`);
   }
 
-  const sourceRPC = NETWORKS[network as Network][chain as ChainName].rpc;
+  const sourceRPC = NETWORKS[network][chain].rpc;
   const sourceChainProvider = new ethers.providers.JsonRpcProvider(sourceRPC);
-  const targetChainProviders = new Map<ChainName, ethers.providers.Provider>();
+  const targetChainProviders = new Map<Chain, ethers.providers.Provider>();
   for (const key in NETWORKS[network]) {
     targetChainProviders.set(
-      key as ChainName,
-      new ethers.providers.JsonRpcProvider(
-        NETWORKS[network as Network][key as ChainName].rpc
-      )
+      key as Chain,
+      new ethers.providers.JsonRpcProvider(NETWORKS[network][key as Chain].rpc)
     );
   }
 
-  const info = await relayer.getWormholeRelayerInfo(chain, argv.tx, {
-    environment: network,
-    sourceChainProvider,
-    targetChainProviders,
-  });
+  // TODO: Convert this over to sdkv2
```

Comment:
This functionality doesn't exist in sdkv2.  To keep this, as is, I would need to convert sdkv2::Chain to sdkv1::ChainName.
I'll see if I can rewrite it locally.

Reply:
We have 2 choices:  1) open an issue and fix in future release, or 2) add a few to several hundred lines of code to this PR in order to implement relayer info functionality.

Reply:
That is extremely unfortunate that `getWormholeRelayerInfo` only takes the `ChainName`. I would prefer to avoid breaking this though. Does sdkv2 provide a method for `ChainToV1ChainName`? If not, it seems like a useful thing for us to shim here for the time being. I'm happy to help whip something up.